size.sh

```diff
@@ -0,0 +1,28 @@
+BASE_PATH=dist/a11y-dialog
+
+# Build the dist files
+npm run build -- --silent
+
+# Gzip files
+gzip -9fkc $BASE_PATH.min.js > $BASE_PATH.min.js.gz
+gzip -9fkc $BASE_PATH.esm.min.js > $BASE_PATH.esm.min.js.gz
+
+# Brotli files
+brotli -9fkc $BASE_PATH.min.js > $BASE_PATH.min.js.br
+brotli -9fkc $BASE_PATH.esm.min.js > $BASE_PATH.esm.min.js.br
+
+# Print sizes
+ls -lh $BASE_PATH.js            | awk '{print "CJS raw size    :", $5"B"}'
+ls -lh $BASE_PATH.min.js        | awk '{print "CJS min size    :", $5"B"}'
+ls -lh $BASE_PATH.min.js.gz     | awk '{print "CJS gzip size   :", $5"B"}'
+ls -lh $BASE_PATH.min.js.br     | awk '{print "CJS brotli size :", $5"B"}'
```

Comment:
```suggestion
ls -lh $BASE_PATH.js            | awk '{print "UMD raw size    :", $5"B"}'
ls -lh $BASE_PATH.min.js        | awk '{print "UMD min size    :", $5"B"}'
ls -lh $BASE_PATH.min.js.gz     | awk '{print "UMD gzip size   :", $5"B"}'
ls -lh $BASE_PATH.min.js.br     | awk '{print "UMD brotli size :", $5"B"}'
```

These are UMD modules, which cover a number of possible module types, including CJS. We never build a CJS module because CJS is a Node-specific spec.